package.json

```diff
@@ -61,5 +61,8 @@
   "engines": {
     "node": ">=8.11",
     "yarn": ">=1.12.3"
+  },
+  "dependencies": {
+    "ovh-ui-angular": "^2.25.1"
```

Comment:
I think the monorepo itself doesn't require `ovh-ui-angular`.

Reply:
This has been removed